content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/clientlibs/site/js/image.js

```diff
@@ -304,6 +362,9 @@
                         if (property && typeof property.transform === "function") {
                             that._properties[key] = property.transform(options[key]);
                         } else {
+                            if (key === "bgsrc") {
```

Comment:
Would `property.transform` ever be set on a BG image?  if so, that would cause this code to not be hit.  Is that okay?